gateway/src/main/kotlin/Intent.kt

```diff
@@ -151,6 +151,9 @@ public sealed class Intent(public val code: DiscordBitSet) {
      */
     public object DirectMessageTyping : Intent(1 shl 14)
 
+    @PrivilegedIntent
+    public object MessageContent : Intent(1 shl 15)
+
```

Comment:
Also update the `values` property in the companion object, it seems to miss `GuildScheduledEvents` too (same order as the declaration would also be nice).

`Intents.nonPrivileged` should now also remove `MessageContent`, I suggest adding `Intents.privileged` and building `Intents.nonPrivileged` with `+Intents.all; -Intents.privileged`

Reply:
I've taken the liberty to do this myself.